packages/interface/package.json

```diff
@@ -19,31 +19,23 @@
     "@ethereum-attestation-service/eas-sdk": "^1.5.0",
     "@hatsprotocol/sdk-v1-core": "^0.10.0",
     "@hookform/resolvers": "^3.3.4",
-    "@nivo/boxplot": "^0.84.0",
     "@nivo/line": "^0.84.0",
-    "@pinata/sdk": "^2.1.0",
-    "@radix-ui/react-accordion": "^1.1.2",
     "@radix-ui/react-dialog": "^1.0.5",
     "@radix-ui/react-dropdown-menu": "^2.0.6",
     "@rainbow-me/rainbowkit": "^2.0.1",
-    "@rainbow-me/rainbowkit-siwe-next-auth": "^0.4.0",
     "@semaphore-protocol/core": "4.0.0-beta.16",
     "@semaphore-protocol/data": "4.0.0-beta.16",
     "@t3-oss/env-nextjs": "^0.8.0",
-    "@tailwindcss/forms": "^0.5.7",
     "@tanstack/react-query": "^5.24.1",
-    "@testing-library/react": "^14.1.2",
```

Comment:
Let's keep this one. It will be useful for tests.

Reply:
is it needed as dependency or can it go in devDependencies?

Reply:
devDeps will be best